crates/l2/contracts/bin/deployer/cli.rs

```diff
@@ -245,8 +243,8 @@ impl Default for DeployerOptions {
             .unwrap(),
             env_file_path: None,
             deposit_rich: false,
-            private_keys_file_path: "../../test_data/private_keys_l1.txt".to_string(),
-            genesis_l1_path: "../../test_data/genesis-l1-dev.json".to_string(),
+            private_keys_file_path: "../../test_data/private_keys_l1.txt".into(),
+            genesis_l1_path: "../../test_data/genesis-l1-dev.json".into(),
```

Comment:
As part as being more production-ready, I'd avoid default values like this one with test data

Reply:
I think both should be optional and `None` as default, as they are both only required if `deposit_rich` is set

Reply:
Done in [586e5ab](https://github.com/lambdaclass/ethrex/pull/2744/commits/586e5ab63a06b022f1bffc54f48eaf3dd214546e).